src/Tribe/Field.php

```diff
@@ -340,6 +341,20 @@ public function doFieldName( $multi = false ) {
 		}
 
 		/**
+		 * returns the field's placeholder
+		 *
+		 * @return string the field value
+		 */
+		public function doFieldPlaceholder() {
```

Comment:
Looks like this is a brand new method ... with that in mind can we use snake_case?